src/wallet/rpc.rs

```diff
@@ -216,98 +151,46 @@ impl Wallet {
             is_first_imported && is_last_imported
         };
```

Comment:
We can remove this whole "is fidelity imported" check.. Just like swapcoins it doesn't hurt a lot to just reimport them if they are imported before. Faster than checking import via two separate rpc calls.

Reply:
Makes sense...  Removed in next commit